ci-operator/step-registry/upi/install/gcp/upi-install-gcp-commands.sh

```diff
@@ -489,9 +489,14 @@ gcloud deployment-manager deployments delete -q "${INFRA_ID}-bootstrap"
 echo "$(date -u --rfc-3339=seconds) - Approving the CSR requests for nodes..."
 function approve_csrs() {
   while true; do
-    oc get csr -ojson | jq -r '.items[] | select(.status == {} ) | .metadata.name' | xargs --no-run-if-empty oc adm certificate approve
-    sleep 15 & wait
-    continue
+    if [[ ! -f /tmp/install-complete ]]; then
```

Comment:
I understand that this is copied from the template, but is there a reason why we wouldn't want to make this the while condition?
```
while [[ ! -f /tmp/installcomplete ]]; do
```

Reply:
It actually makes better sense but it was in the templates and I thought it was there for a reason. I think this change should be fine.